test/test_browser.py

```diff
@@ -3244,6 +3245,7 @@ def test_sdl2_mixer_wav(self, flags):
     # TODO: need to source freepats.cfg and a midi file
     # 'mod': (['mid'],    'MIX_INIT_MID', 'midi.mid'),
   })
+  @no_wasm64('TODO: SDL2 audio memory64')
```

Comment:
I just submitted a PR on the SDL side: https://github.com/libsdl-org/SDL/pull/12332

Can you use that URL as the string argument to `no_wasm64` here.

Reply:
Done.